### ISSUE TITLE
Provide option to disable the border on image as fog

### ISSUE THREAD

**Issue by UranusBytes** [League-of-Foundry-Developers/simplefog#74]
When an image is used as fog layer, a border is draw to show what is visible.  Provide an option to disable this border.  The use case is that the fog layer is a map without labels or other items, but the fog layer is progressively removed to show the labels/whatever on the background.

**Comment by Derinzed** [comment]
Thank you for this, sorry for not getting back to you quickly.

**Comment by UranusBytes** [comment]
This effect is caused by the tint (Player Fog Color) applied to the canvas layer.  If you set the Player Fog Settings Color to a value of `#FFFFFF`  this doesn't actually set the tint to white, but removes the tint effect.  https://pixijs.download/dev/docs/PIXI.Sprite.html#tint

Note that even with the tint set to `#FFFFFF`, any grids added to the map will still only be shown on the unmasked portion of the image, so if a grid line happens to be behind a masked area, players could still deduce that there was masked map content there.


Player Fog Color of `#000000`
<img width="211" alt="image" src="https://user-images.githubusercontent.com/11236319/188270019-a1b52e4a-3137-49a0-b653-afb7b2227fcc.png">

Player Fog Color of `#FFFFFF`
<img width="211" alt="image" src="https://user-images.githubusercontent.com/11236319/188270052-9713030d-effc-4917-8231-509e514cf26e.png">

**Comment by Derinzed** [comment]
I seem to still end up with a white border when using #FFFFFF:

![image](https://user-images.githubusercontent.com/45581121/188280027-f7219ff4-7ca0-4b72-8507-41d15710bca9.png)

Settings:
![image](https://user-images.githubusercontent.com/45581121/188280032-6c66549b-0162-4d40-b93b-c22bbb82cf36.png)

is there another setting that I've configured improperly?

**Comment by UranusBytes** [comment]
No, you are correct.  I was mistaken in the previous testing I'd done.  I can confirm that setting to FFFFFF still shows as white.  I had a code fix prototyped earlier.  Let me resurrect it...

**Comment by UranusBytes** [comment]
Same as #46
